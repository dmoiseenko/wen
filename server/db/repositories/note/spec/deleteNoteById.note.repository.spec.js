const bootstrap = require('../../../bootstrap/bootstrap.js');
const deleteNoteById = require('../deleteNoteById.note.repository.js');
const getNotes = require('../getNotes.note.repository');

beforeEach(() => bootstrap());

it('should delete note by id', async () => {
  await deleteNoteById(1);

  const notes = await getNotes();

  expect(notes).toMatchSnapshot();
});

it('should return deleted note', async () => {
  const actual = await deleteNoteById(1);

  expect(actual).toMatchSnapshot();
});
