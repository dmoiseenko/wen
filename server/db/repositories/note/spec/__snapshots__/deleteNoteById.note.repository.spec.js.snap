// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should delete note by id 1`] = `
Array [
  Object {
    "id": 2,
    "text": "Look deep into nature, and then you will understand everything better. Albert Einstein",
  },
  Object {
    "id": 3,
    "text": "Be yourself; everyone else is already taken. Oscar Wilde",
  },
]
`;

exports[`should return deleted note 1`] = `
Object {
  "id": 1,
  "text": "Well done is better than well said. Benjamin Franklin",
}
`;
